lib/orbital/ext/active_merchant/active_merchant.rb

```diff
@@ -59,6 +59,54 @@ def commit(order, message_type, trace_number=nil)
                          :cvv_result => OrbitalGateway::CVVResult.new(response[:cvv2_resp_code])
                      })
       end
+
+      # A – Authorization request
+      def authorize(money, creditcard, options = {})
+        order = build_new_order_xml(AUTH_ONLY, money, options) do |xml|
+          add_creditcard(xml, creditcard, options[:currency], options[:cvv_indicator_visa_discover])
+          add_address(xml, creditcard, options)
+          if @options[:customer_profiles]
+            add_customer_data(xml, creditcard, options)
+            add_managed_billing(xml, options)
+          end
+        end
+        commit(order, :authorize, options[:trace_number])
+      end
+
+      # AC – Authorization and Capture
+      def purchase(money, creditcard, options = {})
+        order = build_new_order_xml(AUTH_AND_CAPTURE, money, options) do |xml|
+          add_creditcard(xml, creditcard, options[:currency], options[:cvv_indicator_visa_discover])
+          add_address(xml, creditcard, options)
+          if @options[:customer_profiles]
+            add_customer_data(xml, creditcard, options)
+            add_managed_billing(xml, options)
+          end
+        end
+        commit(order, :purchase, options[:trace_number])
+      end
+
+      def add_creditcard(xml, creditcard, currency=nil, cvv_indicator_visa_discover=false)
+        unless creditcard.nil?
+          xml.tag! :AccountNum, creditcard.number
+          xml.tag! :Exp, expiry_date(creditcard)
+        end
+
+        xml.tag! :CurrencyCode, currency_code(currency)
+        xml.tag! :CurrencyExponent, currency_exponents(currency)
+
+        unless creditcard.nil?
+          if %w( visa discover ).include?(creditcard.brand)
+            if cvv_indicator_visa_discover
```

Comment:
Do we want also to let the user pass the actual value (in case we need to set `2` in some cases)?

Something like:

```
xml.tag! :CardSecValInd, (creditcard.verification_value? ? '1' : cvv_indicator_override_visa_discover || '9')
```